docs/source/workflows/index.rst

```diff
@@ -12,16 +12,16 @@ This section is presenting some workflows that involve the usage of multiple SEP
 
     If your workflow is not described here and you think it would make sense to have it in the main SEPAL documentation, please send us a message in our `GitHub issue tracker <https://github.com/openforis/sepal-doc/issues/new/choose>`__.
 
-Table of content
-----------------
+Table of contents
+-----------------
 
 .. toctree::
     :maxdepth: 1
     :glob:
 
-    Sample Based Area estimation<area_estimation>
-    Standard Operating Procedures<sop_template>
-    coded
-    peatland_mapping
-    nrt
-    drivers
+    Sample-based area estimation <area_estimation>
```

Comment:
@alexryangregor  In this case you have a `toctree` directive and inside you have links to the actual page.
You can give a custom name preceed by the `Custom title <name_of_file>`, or just leave the `name` of the file you're referencing and when the documentation is built, it will put the title of that page.

Reply:
As you can see in [ac9fd16](https://github.com/openforis/sepal-doc/pull/363/commits/ac9fd16d11f30f583e76b234561b3e2023132b81), I have updated just with the page names.

Reply:
@dfguerrerom Thank you for explaining this. 

Ideally, documentation page titles should follow sentence case (i.e. capitalizing the first letter of the first word and nothing else; e.g. Peatland mapping) – unless of course there is an acronym in all caps or a proper noun (e.g. BFAST Explorer, SEPAL documentation). 

From a content strategy perspective, consistency is key because it increases clarity for readers, thus improving user experience. While it's still in draft form, I recommend taking a look at [_SEPALSTYLE_](https://github.com/openforis/sepal-doc/blob/437e9ddfe4f292b8a315ee117bc35b8851a5ed52/docs/source/team/style_guide.rst) (the style guide I'm working on for the documentation), which provides guidance on writing and presenting the documentation in general. The style guide should also provide you with a better idea of my rationale behind my decisions while copy-editing : )

**In this case, I think it's more efficient to update the original name of the file, rather than using a custom name.** Does this make sense?

Reply:
that makes sense to me as well, you can use this branch to do any edits you want to do https://github.com/openforis/sepal-doc/pull/396.

Reply:
Noted. Thanks @dfguerrerom!